index.html

```diff
@@ -918,6 +937,37 @@ <h1 class="editableArea">Headline1</h1>
 						</div>
 					</div>
 				</li>
+
+				<li title="arrow">
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M0,0 Q25,75 100 100" marker-start="url(#arrowstart)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M0,0 Q75,25 100 100" marker-end="url(#arrowend)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+				</li>
+				<li title="arrow">
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M100,0 Q25,25 0 100" marker-start="url(#arrowstart)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+					<div class="newMockElement boxarrow">
+						<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none">
+							<path d="M100,0 Q75,75 0 100" marker-end="url(#arrowend)" stroke="black" fill="none" stroke-width="2"/>
+						</svg>
+					</div>
+				</li>
+				<svg class="hideme" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The arrow SVGs reference the 2 marker tags inside the `hideme` SVG by the IDs `arrowstart` and `arrowend` - see [Applying SVG effects to HTML content](https://developer.mozilla.org/en-US/docs/Web/SVG/Applying_SVG_effects_to_HTML_content). The extra `hideme` SVG exists because if it were to be embedded within `newMockElement` then the IDs would be duplicated every time a new arrow was dropped on the page.

I apply the `hideme` class because SVGs often come with a pre-set height.

I hope this helps explain my reasoning! ???